### DISCUSSION TITLE
Please give some love to LORA merging too

### DISCUSSION THREAD

**Discussion by le-khang** [hako-mikan/sd-webui-supermerger#162] [General]
Thank you for this awesome extension Hako Mikan. There are many features and instructions for Model merging so I'd love to have them for LORA merging too, such as MBW, Random merge, Bulk merge, etc.
Thank you again.

**Comment by TomLucidor** [comment]
Seconding this and also Khoya_SS exists for a reason... but then again another pair of eyes don't hurt
